client/state/comments/models/comment.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Internal dependencies
+ */
+import { Author } from './comment-author'; // eslint-disable-line no-unused-vars
+import { CommentInfo } from './comment-info'; // eslint-disable-line no-unused-vars
+
+export function Comment( constructor ) {
+	Object.defineProperty( this, 'constructor', {
+		value: constructor || this
+	} );
+}
+
+export function LoadingComment( siteId, commentId ) {
```

Comment:
is it actually loading? I mean, there's a request in-flight while the instance is of `LoadingComment`?